src/Components/GitApp/Repositories/index.js

```diff
@@ -0,0 +1,30 @@
+import React, {Component} from 'react';
+import {View, Text, FlatList} from 'react-native';
+import {RepoHeader, RepoText} from './styledComponents';
+import Repository from './Repository';
+import UserModel from '../../../Stores/Models/UserModel';
+import UserServices from '../../../Services/UserServices';
+import {observer} from 'mobx-react';
```

Comment:
"3rd party imports
<NEW LINE>
internal lib imports
<NEW LINE>
Group the imports(high to low) based on number of `../` if possible follow alphabetical Order
<NEW LINE>"